policy-service/README.md

```diff
@@ -20,8 +20,22 @@ limitations under the License.
 
 # Policy Service
```

Comment:
Is there any mention of the Hierarchy of rules - rules on a resource also include all rules on that resource's parent (and grand-parents etc.)